test/e2e/operator_test.go

```diff
@@ -2139,6 +2139,141 @@ func TestContainerLogging(t *testing.T) {
 	}
 }
 
+func TestContainerLoggingMaxLength(t *testing.T) {
+	t.Parallel()
+	icName := types.NamespacedName{Namespace: operatorNamespace, Name: "containerloggingmaxlength"}
+	domain := icName.Name + "." + dnsConfig.Spec.BaseDomain
+	ic := newPrivateController(icName, domain)
+	ic.Spec.Logging = &operatorv1.IngressControllerLogging{
+		Access: &operatorv1.AccessLogging{
+			Destination: operatorv1.LoggingDestination{
+				Type: operatorv1.ContainerLoggingDestinationType,
+				Container: &operatorv1.ContainerLoggingDestinationParameters{
+					MaxLength: int32(8192),
+				},
+			},
+			// String with about 8000 characters
+			HttpLogFormat: strings.Repeat("abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789@@@@@@@=", 110) + "8192",
```

Comment:
This log format doesn't make sense, and isn't needed.  `HttpLogFormat` defines how to format the log message.